README.md

```diff
@@ -200,6 +201,21 @@ You can use the Bluesky app to connect to your PDS.
 
 _Note: because the subdomain TLS certificate is created on-demand, it may take 10-30s for your handle to be accessible. If you aren't seeing your first post/profile, wait 30s and try to make another post._
 
+### Setting up SMTP
+
+To be able to verify users' email addresses and send other emails, you need to set up an SMTP server.
+
+One way to do this is to use an email service. [Resend](https://resend.com/) and [SendGrid](https://sendgrid.com/) are two popular choices.
+
+Create an account and API key on an email service and set these variables in `/pds/pds.env` (example with Resend):
+
+```
+PDS_EMAIL_SMTP_URL=smtps://resend:<your api key here>@smtp.resend.com:465/
```

Comment:
Can we specify reccomended locations for these env keys? 
Example:
- a .env file
- in your .bashrc
- other?
I used the out of the box installer script, and am curious where all the resources ended up on my system as well - without that it isn't clear how to inject these .env variables.